conf.d/pure.fish

```diff
@@ -47,6 +47,11 @@ _pure_set_default pure_color_jobs pure_color_normal
 _pure_set_default pure_show_system_time false
 _pure_set_default pure_color_system_time pure_color_mute
 
+# Nix build environment
+_pure_set_default pure_enable_nixdevshell true
```

Comment:
This feature is OS-specific, I don't want it to be enabled by default as it could cause bugs in other OSes.
```suggestion
_pure_set_default pure_enable_nixdevshell false
```

So you will need to edit your `$__fish_config_dir/config.fish` to enable it:

```
set --universal --export pure_enable_nixdevshell true
```

Reply:
Many thanks for your review!
Although Nix package manager is available on any Linux distros and macOS, I agree to disable this option by default as I think it's not so popular.